designer/client/styles/index.scss

```diff
@@ -82,7 +83,9 @@ main {
 
   @include govuk-media-query($from: tablet) {
     .menu {
-      position: fixed;
+      position: -webkit-sticky; /* Safari */
```

Comment:
Better to take care of prefixes in the webpack build? Will ensure they're never missed out 
eg https://github.com/postcss/autoprefixer#webpack

Reply:
yes, i will bring in postcss